Dockerfiles/network-tracer/Dockerfile-ebpf

```diff
@@ -0,0 +1,17 @@
+# Used to compile eBPF code
```

Comment:
Is there a better place we can move these dockerfiles? I'm not sure `/Dockerfiles` is meant for build pipeline specific dockerfiles (may be wrong)

Reply:
👍  I can move them into the `tools` directory under `ebpf`